src/Infrastructure/LeanCode.Serialization/JsonConverters/JsonLaxDateOnlyConverter.cs

```diff
@@ -1,18 +1,57 @@
+using System.Buffers;
 using System.Globalization;
+using System.Text;
+using System.Text.Encodings.Web;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
 namespace LeanCode.Serialization;
 
 public class JsonLaxDateOnlyConverter : JsonConverter<DateOnly>
 {
+    private static readonly UTF8Encoding encoder = new(false);
+    private const int maxBufferSize = 64;
```

Comment:
I think this should be six times the length in chars of the longest valid value because every character could be escaped as `\u0123` sequence.